src/ConfigurationException/InvalidFixerConfigurationException.php

```diff
@@ -20,6 +20,7 @@
  * @author SpacePossum
  *
  * @internal
+ * @final TODO make class final on 3.0
```

Comment:
please mark `TODO` as `@TODO` ;) (so here, you would add 2 new annotations
also, as class is internal, we could make it really final.it was internal since beginning: https://github.com/FriendsOfPHP/PHP-CS-Fixer/commit/87242c89f47a9662d2f413f99e8c4f51b7029785